how to run ACCESS-OM3 beta release (#959)
ACCESS-NRI/access-hive.org.au: docs/models/run-a-model/run-access-om2.md

If you are unsure whether {{ model }} is the right choice for your experiment, take a look at the overview of [ACCESS Models](/models).

> All {{{ model }}} configurations are open source, licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/?ref=chooser-v1")![CC icon](https://mirrors.creativecommons.org/presskit/icons/cc.svg?ref=chooser-v1){: style="height:1em;margin-left:0.2em;vertical-align:text-top;"}![BY icon](https://mirrors.creativecommons.org/presskit/icons/by.svg?ref=chooser-v1){: style="height:1em;margin-left:0.2em;vertical-align:text-top;"} and available on [ACCESS-NRI GitHub]({{github_configs}}).

Reviewer: ```suggestion
All {{model}} configurations are open source, licensed under [CC BY 4.0](https://creativecommons.org/licenses/by/4.0/?ref=chooser-v1")![CC icon](https://mirrors.creativecommons.org/presskit/icons/cc.svg?ref=chooser-v1){: style="height:1em;margin-left:0.2em;vertical-align:text-top;"}![BY icon](https://mirrors.creativecommons.org/presskit/icons/by.svg?ref=chooser-v1){: style="height:1em;margin-left:0.2em;vertical-align:text-top;"} and available on [ACCESS-NRI GitHub]({{github_configs}}).
```

### Start the run from a specific restart file {: id='specific-restart'}

> By default, the configuration will start from a "cold-start", where initial conditions are set based on observations of salinity and temperature, but all other model variables are 0.

Reviewer: but is correct here

!!! warning
    In some cases, if the supplied restart file is not fully compatible with the model configuration, experiments using a custom restart file may require additional manual adjustments to run correctly.

!!! warning
    The restart flag used here will only be applied if there is no restart directory in archive, and so does not have to be removed for subsequent submissions. See [Payu docs](https://payu.readthedocs.io/en/stable/config.html#miscellaneous) for further details.

>     <br><br>

Reviewer: ```suggestion
```

!!! warning
    In some cases, if the supplied restart file is not fully compatible with the model configuration, experiments using a custom restart file may require additional manual adjustments to run correctly.

!!! warning

>     The restart flag used here will only be applied if there is no restart directory in archive, and so does not have to be removed for subsequent submissions. See [Payu docs](https://payu.readthedocs.io/en/stable/config.html#miscellaneous) for further details.

Reviewer: ```suggestion
    The restart option used here will only be applied if there is no restart directory in archive, and so does not have to be removed for subsequent submissions. See [Payu docs](https://payu.readthedocs.io/en/stable/config.html#miscellaneous) for further details.
```

>     The restart flag used here will only be applied if there is no restart directory in archive, and so does not have to be removed for subsequent submissions. See [Payu docs](https://payu.readthedocs.io/en/stable/config.html#miscellaneous) for further details.

Reviewer: Its not only the `--restart` flag, it can be `-r` only `restart:` manually added to the config file. I'll change `flag` to `option`

>     The restart flag used here will only be applied if there is no restart directory in archive, and so does not have to be removed for subsequent submissions. See [Payu docs](https://payu.readthedocs.io/en/stable/config.html#miscellaneous) for further details.

Reviewer 2: Yes, I think for om3 below there is the `-r` mention as well, in general I would have the same phrase whatever you choose. I like `option`.

project: ol01
```

To use a `/scratch` storage allocation other than `project` (or your default if `project` is not set) then also set `shortpath` (e.g. to the desired `/scratch/PROJECT_CODE`). 

!!! warning

>     If changing projects during an experiment, set the `shortpath` field so that it's consistent for all runs of an experiment.<br>

Reviewer: ```suggestion
    If changing projects during an experiment, set the `shortpath` field so that it's consistent for all runs of an experiment.
```

    Doing this will make sure the same `/scratch` location is used for the _laboratory_, regardless of which project is used to run the experiment.
    <br><br>

>     To run {{ model }}, you need to be a member of a project with allocated _Service Units (SU)_. For more information, check [how to join relevant NCI projects](/getting_started/set_up_nci_account#join-relevant-nci-projects).

Reviewer: ```suggestion
```

To use a `/scratch` storage allocation other than `project` (or your default if `project` is not set) then also set `shortpath` (e.g. to the desired `/scratch/PROJECT_CODE`). 

!!! warning
    If changing projects during an experiment, set the `shortpath` field so that it's consistent for all runs of an experiment.
    Doing this will make sure the same `/scratch` location is used for the _laboratory_, regardless of which project is used to run the experiment.
    <br><br>

>     To run {{ model }}, you need to be a member of a project with allocated _Service Units (SU)_. For more information, check [how to join relevant NCI projects](/getting_started/set_up_nci_account#join-relevant-nci-projects).

Reviewer: ok done

project: ol01
```

> To use a `/scratch` storage allocation other than `project` (or your default if `project` is not set) then also set `shortpath` (e.g. to the desired `/scratch/PROJECT_CODE`). 

Reviewer: ```suggestion
For model configurations and output to be saved to a `/scratch` storage allocation other than `project` (or your default if `project` is not set) then also set `shortpath` (e.g. to the desired `/scratch/PROJECT_CODE`). 
```

!!! warning

>     If changing projects during an experiment, set the `shortpath` field so that it's consistent for all runs of an experiment.

Reviewer: These settings are for where model output is saved, not accessing data in other projects. I'll change:

_To use a `/scratch` storage ..._ 

to 

_For model configurations and output to be saved to a `/scratch` storage ..._

> project: ol01

Reviewer 3: Edit: in above line `For example, to run ... replace `PROJECT_CODE` wih `ol01`
Typo: `with'`

      - '*.nc.*'
      - 'iceh.????-??-??.nc'
```

> To enable syncing, change `enable` to `True`, and set `path` to a location on `/g/data`, where _payu_ will copy output and restart folders. A sensible `path` could be: `/g/data/$PROJECT/$USER/{{{ model }}}/experiment_name/`.

Reviewer: ```suggestion
To enable syncing, change `enable` to `True`, and set `path` to a location on `/g/data`, where _payu_ will copy output and restart folders. A sensible `path` could be: `/g/data/$PROJECT/$USER/{{model}}/experiment_name/`.
```

!!! admonition tip

>     The {{{ model }}} default configurations include a [userscript](#userscripts) in the _sync_ step that concatenates daily history/diagnostic output from the Sea-Ice model (CICE5) into monthly files. This speeds up access and saves storage space, but will only run if _sync_ is enabled.

Reviewer: ```suggestion
    The {{model}} default configurations include a [userscript](#userscripts) in the _sync_ step that concatenates daily history/diagnostic output from the Sea-Ice model (CICE5) into monthly files. This speeds up access and saves storage space, but will only run if _sync_ is enabled.
```

To enable syncing, change `enable` to `True`, and set `path` to a location on `/g/data`, where _payu_ will copy output and restart folders. A sensible `path` could be: `/g/data/$PROJECT/$USER/{{model}}/experiment_name/`.

!!! admonition tip
    The {{model}} default configurations include a [userscript](#userscripts) in the _sync_ step that concatenates daily history/diagnostic output from the sea ice model (CICE5) into monthly files. This speeds up access and saves storage space, but will only run if _sync_ is enabled.

### Pruning model restarts

> By default, {{ model }} saves restart files after each run, allowing subsequent simulations to resume from a previously saved model state. The default {{ model }} run length and restart period can be changed (see [Change run length and restart period](#change-run-length-and-restart-period)).<br>
> However, restart files can occupy significant disk space, and keeping all of them at the end of a whole experiment is often not necessary. If disk space is limited, consider using _payu_'s restart files pruning feature. This allows you to keep all the restart files during an experiment (constituted by multiple runs) in case of a crash, while reducing disk usage after the whole experiment completes by automatically removing redundant restart files.

Reviewer 2: After our chat about the pruning feature, this is not entirely correct. 
I would rephrase to something like:

```suggestion
By default, {{ model }} saves restart files after each run, allowing subsequent simulations to resume from a previously saved model state. The default {{ model }} run length and restart period can be changed (see [Change run length and restart period](#change-run-length-and-restart-period)).<br>
However, restart files can occupy significant disk space, and keeping all of them throughout an entire experiment is often not necessary. If disk space is limited, consider using _payu_'s restart files pruning feature, controlled by the `restart_freq` field of the `config.yaml`.
By default, every `restart_freq` _payu_ removes intermediate restart files, keeping only: 
- the two most recent restarts
- restarts corresponding to the `restart_freq` interval
For example, a `restart_freq` set to `1YS` would keep the restart files at the end of each model year, whereas `restart_freq` set to `5YS` would keep those at the end of every fifth model year.
This approach helps reduce disk space while maintaining useful restart points across long experiments, especially useful in case of unexpected crashes.
```

> _Payu_ is configured to prune the number of restart files kept. By default, _payu_ will only keep most recent couple of restarts and the restarts which match the frequency set in the `restart_freq` field of `config.yaml`. This is commonly set to `1YS` to keep one restart per year, or `5YS` for a restart every fifth year. 

Reviewer 2: ```suggestion
```

> The `restart_freq` field in the `config.yaml` file specifies a strategy for retaining restart files.<br>
> This can either be a number (in which case every _nth_ restart file is retained), or one of the following pandas-style datetime frequencies:

Reviewer 2: ```suggestion
The `restart_freq` field in the `config.yaml` can either be a number (in which case every _nth_ restart file is retained), or one of the following pandas-style datetime frequencies:
```
